File: sdkwork-app-sdk-kotlin/src/main/kotlin/com/sdkwork/app/RolloutConfigVO.kt
package com.sdkwork.app

data class RolloutConfigVO(
    val createdAt: String? = null,
    val updatedAt: String? = null,
    val rolloutKey: String? = null,
    val rolloutName: String? = null,
    val config: Map<String, Any>? = null,
    val status: String? = null
)

File: sdkwork-app-sdk-python/sdkwork_app_sdk/models/rollout_config_vo.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class RolloutConfigVO:
    """发布配置响应"""
    created_at: str = None
    updated_at: str = None
    rollout_key: str = None
    rollout_name: str = None
    config: Dict[str, Any] = None
    status: str = None
